Makefile

```diff
@@ -45,6 +49,21 @@ docs: docs-clean ## Runs livereload environment for local documentation editing
 # Spins up livereload environment for editing; blocks.
 	sphinx-autobuild docs/ docs/_build/html
 
+.PHONY: build-app
+build-app: ## Packages Electron app locally via webpack for development and testing
+	npm run build-app
+
+.PHONY: test-unit
```

Comment:
Same as above: for both `test-unit` and `test-2e2`, `npm install` needs to fetch and install libraries on first install or after `make clean`